.travis.yml

```diff
@@ -0,0 +1,89 @@
+sudo: required
+dist: trusty
+
+branches:
+  except:
+  #- l10n_dev
+
+language: node_js
+node_js:
+  - "8"
+  
+addons:
+apt:
+  sources:
+    - google-chrome
+  packages:
+    - google-chrome-stable
+    - google-chrome-beta
+
+env:
+  - TRAVIS_NODE_VERSION="7.9.0"
+cache:
+   directories:
+     - $(npm root -g)
+     - node_modules
+
+notifications:
+  webhooks:
+    urls:
+      - https://webhooks.gitter.im/e/063276d3941fa8443947
```

Comment:
I think we can use private environment variable and hiding our notifications link. We don't want some people spamming our channel right? https://docs.travis-ci.com/user/environment-variables/

Reply:
@mappuji thanks I think we should open an extra/new issue for all our `.travis.yml`es ;)

/me thinks we have at least 4 ...